integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/tpcds/TpcdsLikeSpark.scala

```diff
@@ -146,573 +148,596 @@ object TpcdsLikeSpark {
       appendDat))
   }
 
-  def setupAllCSV(spark: SparkSession, basePath: String, appendDat: Boolean = true): Unit = {
-    tables.foreach(_.setupCSV(spark, basePath, appendDat))
+  def setupAllCSV(
+      spark: SparkSession,
+      basePath: String,
+      useDecimalType: Boolean = false,
```

Comment:
Why place the new parameter here?  Anyone who was invoking the old method now is passing a boolean value that they _thought_ meant `appendDat` but now means `useDecimalType` which seems unfortunate.  If we placed the new parameter at the end of the parameter list then it becomes backwards-compatible and no surprises.

Reply:
Good point. Thanks. I have updated this.